docker/policies/apt_get_missing_yes_flag_to_avoid_manual_input.rego

```diff
@@ -0,0 +1,75 @@
+package appshield.dockerfile.DS021
+
+import data.lib.common
+import data.lib.docker
+
+__rego_metadata__ := {
+	"id": "DS021",
+	"title": "APT-GET Missing '-y' To Avoid Manual Input",
+	"version": "v1.0.0",
+	"severity": "HIGH",
+	"type": "Dockerfile Security Check",
+	"description": "Check if apt-get calls use the flag -y to avoid user manual input.",
+	"recommended_actions": "Add -y flag to apt-get",
+	"url": "https://docs.docker.com/engine/reference/builder/#run",
+}
+
+__rego_input__ := {
+	"combine": false,
+	"selector": [{"type": "dockerfile"}],
+}
+
+short_flags := `(-[^ya..zA..Z]*y[^ya..zA..Z]*)`
```

Comment:
What does `a..z` mean? Not `a-z`? Even if you mean `a-z`, `[^ya-zA-Z]` is probably buggy. I think it excludes `y`, `a-z`, `A-Z`.

Reply:
changed to 
`[a-xzA-XZ]`